moochub-schema.json

```diff
@@ -1,555 +1,3105 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema",
-  "$id": "https://github.com/moochub/schema/releases/tag/v2.3",
+  "$id": "https://github.com/moochub/schema/releases/tag/v3.0",
   "title": "JSON schema for MOOC providers integrated into moochub.org",
   "description": "This schema specifies the JSON format each MOOC provider must offer in order to be integrated into moochub.org",
   "type": "object",
   "properties": {
-    "links": {
```

Comment:
No pagination? 🤔

Reply:
Pagination restored; links and data attributes exist again;